mobile/src/validation-schemas/chat/chat-message-create.validation-schema.ts

```diff
@@ -0,0 +1,20 @@
+import { getNameOf } from 'guruhub-shared/helpers/helpers';
+import Joi from 'joi';
+
+import {
+  ChatValidationMessage,
+  ChatValidationRule,
+} from '~/common/enums/enums';
+import { ChatMessageCreateRequestBodyDto } from '~/common/types/types';
+
+const chatMessageCreate = Joi.object({
+  [getNameOf<ChatMessageCreateRequestBodyDto>('message')]: Joi.string()
```

Comment:
If we use only message in form that we should have another interface that have only message as it's done for note, if we use all fields from ChatMessageCreateRequestBodyDto then it's better to use backend schema

Reply:
But then I need to add trim and min length there?

Reply:
There you have 2 options, if you're going to have all values in form then you don't need separate validation schéma only for message, or you can store in form only message field and use separate schema that contain only this field and then when you pass payload to action (in handler for form submit)  you can add other fields that your request requires